index.md

```diff
@@ -1,12 +1,16 @@
 # Anbox Cloud documentation
 
-Anbox Cloud enables running Android apps on any cloud platform at scale. It uses system containers or virtual machines to run the nested Android containers and [Juju](https://juju.is/) for deployment in a cloud environment.
+Anbox Cloud runs Android in the cloud using lightweight LXD system containers or full virtual machines. Built on Ubuntu, it provides a scalable platform to deploy, manage, and stream Android workloads across public or private infrastructure. Compared to other Android emulation solutions, Anbox Cloud can provide at least twice the density and can serve up to **100 Android instances per server**.
```

Comment:
I'm not sure if we should include this sentence here without providing any hardware specifications for the server to support `100 Android instances per server` , or a reference to support the claim that Anbox Cloud offers "at least twice the density." It currently feels more like a marketing statement.